pkg/cache/application_test.go

```diff
@@ -559,6 +559,7 @@ func TestGetNonTerminatedTaskAlias(t *testing.T) {
 	task1 := NewTask(taskID1, app, context, pod1)
 	app.taskMap[taskID1] = task1
 	task1.sm.SetState(TaskStates().Pending)
+	//nolint:funlen
```

Comment:
Could you please move this suppression to line#527?

Reply:
OK!!